package.json

```diff
@@ -59,6 +62,8 @@
     "s3": "gulp s3",
```

Comment:
found the answer to my own question: https://github.com/mozilla/teach.webmaker.org#generating-a-static-site :grin: